src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php

```diff
@@ -426,4 +426,25 @@ protected function compileEndPushIf()
     {
         return '<?php $__env->stopPush(); endif; ?>';
     }
+
+    /**
+     * Compile the "filled" statements into valid PHP.
+     *
+     * @param  string  $expression
+     * @return string
+     */
+    public function compileFilled($expression)
+    {
+        return "<?php if (\Illuminate\Support\Arr::filled{$expression}): ?>";
```

Comment:
The class `Illuminate\Support\Arr` has no `filled()` method.

The `filled()` helper is added to the global namespace.

```php
return "<?php if (filled{$expression}): ?>";
```